app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -43,10 +46,10 @@
 
     <div class="govuk-grid-row covid__page-header-guidance-section">
```

Comment:
I think you could drop the `govuk-grid-row` from this element and fold it into the two thirds column that precedes it?

Reply:
done